javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
Thanks for the clarification. Looks good to me.

We should still merge it to the official THREE.js repo. Eventually it would be good idea to only maintain the Draco loader there, but let's keep it duplicate for now.